ui/main/src/app/modules/usercard/usercard.component.ts

```diff
@@ -203,8 +203,9 @@ export class UserCardComponent implements OnDestroy, OnInit {
                         const _i18nPrefix = process.id + '.' + process.version + '.';
                         const label = process.name ? (_i18nPrefix + process.name) : process.id;
                         const processToShow = { value: process.id, label: label };
-                        this.processOptions.push(processToShow);
-                        this.loadStatesForProcess(process);
+                        
+                        // Add process option only if there is at least one state
+                        if (this.loadStatesForProcess(process)) this.processOptions.push(processToShow);
```

Comment:
What is the advantage of returning a boolean rather than testing this.statesPerProcesses.get(process.id) for existence and length directly?